tools/wheel_resolver/wheel.py

```diff
@@ -74,4 +75,16 @@ def _is_wheel(url: str) -> bool:
 
 
 def _is_compatible(url: str, tags: typing.List[str]) -> bool:
-    return any(t in url for t in tags)
+    url_tags = extract_wheel_tags(url)
+    return any(t in url_tags for t in tags)
+
+
+def extract_wheel_tags(url: str) -> list[str]:
+    _, _, interpreter, abi, platform = url.replace(".whl", "").split("/")[-1].split("-")
+    interpreters, abis, platforms = (
```

Comment:
I feel like these would be better written just one on each line, that takes three lines instead of five

Reply:
Fair, updated